BENCHMARKS.md

```diff
@@ -9,14 +14,15 @@ Running benchmarks as follows in order:
  maxcdn.purge('ZONEID')
  maxcdn.purge('ZONEID', 'FILE')
  maxcdn.purge('ZONEID', [ 'FILE1','FILE2' ])
- 
+
        user     system      total        real
-get   :  0.010000   0.000000   0.010000 (  1.356269)
-get   :  0.010000   0.000000   0.010000 (  0.911275)
-get   :  0.010000   0.000000   0.010000 (  1.069133)
-post  :  0.000000   0.000000   0.000000 (  6.513997)
-put   :  0.010000   0.000000   0.010000 (  0.831379)
-delete:  0.010000   0.000000   0.010000 (  1.248739)
-purge :  0.010000   0.000000   0.010000 (  4.019390)
-purge :  0.010000   0.000000   0.010000 (  6.221071)
-purge :  0.010000   0.000000   0.010000 (  2.496699)
+get   :  0.010000   0.000000   0.010000 (  0.444660)
+get   :  0.000000   0.000000   0.000000 (  0.180768)
+get   :  0.000000   0.010000   0.010000 (  0.556110)
+post  :  0.010000   0.000000   0.010000 (  3.665597)
+put   :  0.000000   0.000000   0.000000 (  0.370504)
+delete:  0.010000   0.000000   0.010000 (  2.557845)
+purge :  0.000000   0.000000   0.000000 (  1.392236)
```

Comment:
@jdorfman Some seeing some really nice performance updates here. I don't know if that's due to the removal of `curbfu` or because of performance enhancements to the API, but either way... good stuff!